HIP/hip-0000.md

```diff
@@ -0,0 +1,93 @@
+---
+hip: <HIP number (assigned by the HIP editor), usually the PR number>
+title: Add the `file_hash` field to the `/api/v1/blocks` endpoints' results
+author: Giuseppe Bertone <giuseppe.bertone@hashgraph.com>
+working-group: Giuseppe Bertone <giuseppe.bertone@hashgraph.com>
+requested-by: Giuseppe Bertone <giuseppe.bertone@hashgraph.com>
```

Comment:
The requested by needs to be an external user needing this change. I don't think we should move forward without an actual user requiring this.